https://discourse.onlinedegree.iitm.ac.in/t/164277

<a class="mention" href="/u/23f2003413">@23f2003413</a></p><hr>

<h4>23f2001286 (#500)</h4>
<p>sir I am noticing a pattern, that when I am running the datagen first. And then using the evaluate.py, then I am getting the A2 right.<br/>
But running the evaluation.py for the 2nd time cause the A2 to fail…<br/>
Probabbly Because the file in the data folder gets upated should I worry for that…</p><hr>

<h4>jkmadathil (#499)</h4>
<p>New token assigned to the students.